15141135 SUNBT4802301 xscreensaver should not tell root to "xhost +localhost"

Upstream's opinion differs from our requirements here.
---
 driver/demo-Gtk.c |   11 +----------
 1 files changed, 1 insertions(+), 10 deletions(-)

diff --git a/driver/demo-Gtk.c b/driver/demo-Gtk.c
--- a/driver/demo-Gtk.c
+++ b/driver/demo-Gtk.c
@@ -1058,16 +1058,7 @@ await_xscreensaver (state *s)
         strcat (buf, STFU
 	  _("You are running as root.  This usually means that xscreensaver\n"
             "was unable to contact your X server because access control is\n"
-            "turned on.  Try running this command:\n"
-            "\n"
-            "                        xhost +localhost\n"
-            "\n"
-            "and then selecting `File / Restart Daemon'.\n"
-            "\n"
-            "Note that turning off access control will allow anyone logged\n"
-            "on to this machine to access your screen, which might be\n"
-            "considered a security problem.  Please read the xscreensaver\n"
-            "manual and FAQ for more information.\n"
+            "turned on.\n"
             "\n"
             "You shouldn't run X as root. Instead, you should log in as a\n"
             "normal user, and `su' as necessary."));
